database.dbml

```diff
@@ -0,0 +1,28 @@
+Table PostReaction {
+  id int [not null, pk, increment]
+  post_id uuid [not null, unique, ref: > Post.id]
+  user_id uuid [not null, unique, ref: > User.id]
+  comment_id uuid [null, unique, ref: > User.id]
+  created_at timestamp
+}
+
+
+Table Notification {
+  id uuid [not null, pk, unique]
+  created_at timestamp [not null]
+  is_read bool [not null]
+  user_id uuid [not null, ref: > User.id]
+  message_id uuid [null, ref: > Message.id]
+  comment_id uuid [null, ref: > PostComment.id]
+  reaction_id uuid [null, ref: > PostReaction.id]
+  follower_id uuid [null, ref: > Followers.id]
+
+}
```

Comment:
- Use uuid for all primary keys.
- Add default for `is_read`. Also change the name to `status` and have it mapped to an enum of [read, unread].
- change `message_id` to just `message` which should be a string.
- Remove `comment_id`, `reaction_id` and `follower_id` fields.